boards/arm/wio_terminal/wio_terminal_defconfig

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Joel Guittet
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_SAMD51=y
+CONFIG_SOC_PART_NUMBER_SAMD51P19A=y
+CONFIG_SOC_ATMEL_SAMD5X_XOSC32K=y
+CONFIG_SOC_ATMEL_SAMD5X_XOSC32K_AS_MAIN=y
+CONFIG_BOARD_WIO_TERMINAL=y
+CONFIG_ARM_MPU=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_BOOTLOADER_BOSSA=y
+CONFIG_BOOTLOADER_BOSSA_ADAFRUIT_UF2=y
+CONFIG_BUILD_OUTPUT_UF2=y
```

Comment:
Move BOSSA config to a specific section like Console. This could be good for future references.